autotest/utilities/test_gdal_grid.py

```diff
@@ -887,6 +887,52 @@ def test_gdal_grid_clipsrc():
     assert not (cs == 0 or cs == 4672), 'bad checksum'
     ds = None
 
+###############################################################################
+# Test -tr
+
+
+def test_gdal_grid_tr():
+    if gdal_grid is None:
+        pytest.skip()
+
+    #################
+    outfiles.append('tmp/grid_count_70_70.tif')
+    try:
+        os.remove(outfiles[-1])
+    except OSError:
+        pass
+
+    # Create a GDAL dataset from the values of "grid.csv".
+    gdaltest.runexternal(gdal_grid + ' -txe 440720.0 441920.0 -tye 3751320.0 3750120.0 -tr 60 60 -ot Byte -l grid -a count:radius1=70.0:radius2=70.0:angle=0.0:min_points=0:nodata=0.0 data/grid.vrt ' + outfiles[-1])
+
+    # We should get the same values as in "ref_data/grid_count_70_70.tif"
+    ds = gdal.Open(outfiles[-1])
+    ds_ref = gdal.Open('ref_data/grid_count_70_70.tif')
+    assert ds.GetRasterBand(1).Checksum() == ds_ref.GetRasterBand(1).Checksum(), \
+        ('bad checksum : got %d, expected %d' %
+              (ds.GetRasterBand(1).Checksum(), ds_ref.GetRasterBand(1).checksum_ref))
+    ds_ref = None
+    ds = None
+
+    #################
+    outfiles.append('tmp/grid_count_300_300.tif')
```

Comment:
I don't think that this second part of the test is actually needed. It just tests different values of the radius1/radius2 parameters, which are not impacted by this PR

Reply:
That's right. I didn't realise that. Fixed in a488626